### ISSUE TITLE
increased peripheral nerve cross sectional area

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10783]
**Preferred term label:**
increased peripheral nerve cross sectional area

**Synonyms**
Ultrasonography showing increased cross sectional area (CSA) of a peripheral nerve as it travels through a potential site of compression; nerve swelling; nerve enlargement

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Carpal tunnel 
Ulnar tunnel
Cubital tunnel
Radial tunnel

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001147
- label: Increased peripheral nerve cross sectional area
- parents: [Abnormal peripheral nervous system morphology (HP:0000759)](https://hpo.jax.org/app/browse/term/HP:0000759)
- pmid: 
- def: Ultrasonography showing increased cross sectional area (CSA) of a peripheral nerve as it travels through a potential site of compression.
- comment: This feature may be observed in conditions such as carpal tunnel syndrome, ulnar tunnel syndrome, cubital tunnel syndrome, and radial tunnel syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10783
- synonyms: Peripheral nerve swelling(NONE)|Peripheral nerve enlargement(NONE)
